docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3"
```

Comment:
@liquidz `docker-compose.yml` is already in [dev-resources](https://github.com/toyokumo/toyokumo-commons/blob/main/dev-resources/docker-compose.yml),  which is used in Makefile.
Is it for a different purpose?

Reply:
@nsyee Thanks for your confirmation!
Oh, I've missed the file.
I removed this file.